lib/opensusebasetest.pm

```diff
@@ -190,15 +190,15 @@ sub select_bootmenu_more {
     # after installation-images 14.210 added a submenu
     if ($more && check_screen "inst-submenu-more", 1) {
         if (get_var('OFW')) {
-            send_key_until_needlematch 'inst-onmore', 'up';
+            send_key_until_needlematch('inst-onmore', 'up', 10, 5);
```

Comment:
What's about then combining it with the function call below by just changing the key. E.g.:
```perl
send_key_until_needlematch('inst-onmore', get_var('OFW') ? 'up' : 'down, 10, 5);
```
And same for the second call. Just a thought.